src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayPartKind.cs

```diff
@@ -66,7 +66,9 @@ public enum SymbolDisplayPartKind
         /// <summary>The name of a type parameter.</summary>
         TypeParameterName = 26,
         /// <summary>The name of a query range variable..</summary>
-        RangeVariableName = 27
+        RangeVariableName = 27,
+        /// <summary>The nullability of the symbol.</summary>
+        Nullability = 28
```

Comment:
Probably should just use Punctuation, this isn't necessary.

Reply:
I somewhat agree.  Things like `*` and `?` and `[]` are already included today and don't need anything special.